mandatory/2-function-creation.js

```diff
@@ -45,7 +88,11 @@ function canRegister(age) {}
   )
 */
 
-function countReverse(number) {}
+function countReverse(number) {
```

Comment:
I like this decrementing solution. I think the example solution uses a while loop, but I prefer yours! ;)